tests/pos/i4414.scala

```diff
@@ -1,6 +1,7 @@
 import scala.quoted._
 
 object Test {
+  given as QuoteContext = ???
```

Comment:
It's not obvious to me why this is needed, given that all implicits are available and type check, and there are no quotes nor splices.

Reply:
The calls to `a` and `b` synesize some quoted types `[...]`, those require a context.